business.html

```diff
@@ -43,9 +43,9 @@
 </ul>
 
 <section id="scale-features">
-  <h2>Sandstorm + Scale</h2>
+  <h2>Sandstorm for Enterprise</h2>

```

Comment:
Hmm, we've been calling it "Sandstorm Enterprise", not "Sandstorm for Enterprise". Do you think "Sandstorm for Enterprise" is better? I think adding just "Enterprise" as a suffix is more common but I could be convinced either way.